cmake/onnxruntime_framework.cmake

```diff
@@ -63,6 +63,11 @@ if (onnxruntime_ENABLE_TRAINING)
   target_include_directories(onnxruntime_framework PRIVATE ${DLPACK_INCLUDE_DIR})
 endif()
 onnxruntime_add_include_to_target(onnxruntime_framework onnxruntime_common onnx onnx_proto ${PROTOBUF_LIB} flatbuffers)
+
+if (onnxruntime_USE_MIMALLOC)
+    target_link_libraries(onnxruntime_framework mimalloc-static)
```

Comment:
First, after "onnxruntime_framework", you should add "PRIVATE" or "PUBLIC" keyword. 
```
target_link_libraries(onnxruntime_framework PRIVATE mimalloc-static)
```

However, I suggest you add it to https://github.com/microsoft/onnxruntime/blob/master/cmake/CMakeLists.txt#L938 instead. To make sure they are in order. https://github.com/microsoft/onnxruntime/blob/master/docs/cmake_guideline.md#dont-call-target_link_libraries-on-static-libraries